spyder/widgets/fileswitcher.py

```diff
@@ -272,6 +272,8 @@ def __init__(self, parent, plugin, tabs, data, icon):
         self.setWindowOpacity(0.95)
         self.edit.installEventFilter(self.filter)
         self.edit.setValidator(regex_validator)
+        self.edit.setPlaceholderText(_("Write some text to select among the "
+                                       "currently opened files or consoles."))
```

Comment:
``Start typing the name of an open file or console to switch to it.``

Reply:
I totally agree with CAM's suggestion here. Mine was very lousy, sorry.

@juanis2112, please address his review and this will be ready.

Reply:
Haha, it wasn't all that lousy. There was just a little better way to put it in my opinion.

Reply:
It was! I mean, I didn't mean to put that text directly, just something like it but well phrased ;-p